### PR TITLE
feat(Loader):Added Loader in All pages

### PR DESCRIPTION
**Pull request by sandeep-v1404**
Signed-off-by: sandeep-v1404 <valpasanisandeep@gmail.com>

### Functionality:
Added Loader to all Pages
### Solution:
To create a Loader Component

### Risk level:
- [ ] high 
- [X] medium
- [ ] low

### How to test:
1. ng serve
2. Dashboard , Tasks ,TaskDetails Contains a Loader
3. Loader after Clicking Submit in CreateNewTask ,CreateNewSprint ,EditTask ,LogWork

### CODE REVIEW

File: worktrolly-ui/src/app/body/create-new-session/create-new-session.component.ts [hunk 1]
```diff
@@ -35,10 +35,11 @@ export class CreateNewSessionComponent implements OnInit {
     var mm = String(today.getMonth() + 1).padStart(2, '0');
     var yyyy = today.getFullYear();
 
-    this.todayDate = dd+"/"+mm+"/"+yyyy;
+    this.todayDate = dd + "/" + mm + "/" + yyyy;
   }
 
   async createNewSession() {
+    this.enableLoader = true;
```
Comment (shivamgoyal7) [inline worktrolly-ui/src/app/body/create-new-session/create-new-session.component.ts:42]: After completion of the request make this as a false.

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]
Add loader = false in every catch method, because if any error will occur we need to terminate the loader.

**Review by shivamgoyal7 — APPROVED** [review]
ship it

**Review by viv9k — APPROVED** [review]
ship it

**Pull request merged**